internal/provider/resource_tfe_registry_module.go

```diff
@@ -104,6 +116,18 @@ func resourceTFERegistryModule() *schema.Resource {
 					true,
 				),
 			},
+			"test_config": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"tests_enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
```

Comment:
I'm wondering if there would ever be a rationale for having a test_config block but with a `tests_enabled` attribute of `false`. I'm sort of in the dark without API docs. Can you describe what tests_enabled does and when it might change?